Lib/test/test_sqlite3/test_factory.py

```diff
@@ -240,6 +240,14 @@ def test_sqlite_row_as_sequence(self):
         self.assertEqual(list(reversed(row)), list(reversed(as_tuple)))
         self.assertIsInstance(row, Sequence)
 
+    def test_sqlite_row_contains(self):
+        """Checks if the row object can be used with `in`"""
```

Comment:
FYI, these kind of comments result in weird `unittest` output in verbose mode; it is better to use verbose test names :)

Reply:
TIL! I was falling the style of other test cases in this file.

Reply:
I misremembered! :) It is _docstrings_ that must be avoided. Ordinary comments are quite ok.